integration-tests/auth_utils.go

```diff
@@ -33,35 +27,26 @@ func authenticate(httpClient *http.Client, identity identity, userID string, pin
 	// Get V (used in /pass2) param using Y param from the pass1 response.
 	V, err := wrap.Client2BN254CX(xR, hex2bytes(p1Response.Y), S)
 	if err != nil {
-		return nil, fmt.Errorf("error getting client 2: %w", err)
+		return fmt.Errorf("error getting client 2: %w", err)
 	}
 
 	// Call to /rps/v2/pass2 endpoint.
-	qrURL, err := url.Parse(authorizeResponse.QRURL)
-	if err != nil {
-		return nil, fmt.Errorf("error parsing the QR URL: %w", err)
-	}
-	p2Response, err := pass2Request(httpClient, identity, V, qrURL.Fragment)
+	p2Response, err := pass2Request(httpClient, identity, V, code)
 	if err != nil {
-		return nil, fmt.Errorf("error making the pass 1 request: %w", err)
+		return fmt.Errorf("error making the pass 2 request: %w", err)
 	}
 
 	// Call to /rps/v2/authenticate endpoint.
 	authResponse, err := authenticateRequest(httpClient, p2Response.AuthOTT)
 	if err != nil {
-		return nil, fmt.Errorf("error making the authenticate request: %w", err)
-	}
-	if authResponse.Status != 200 {
-		return nil, fmt.Errorf("error making the authenticate request, status: %v", authResponse.Status)
+		return fmt.Errorf("error making the authenticate request: %w", err)
 	}
 
-	// Call to /rps/v2/access endpoint.
-	accessResponse, err = accessRequest(httpClient, authorizeResponse.WebOTT)
-	if err != nil {
-		return nil, fmt.Errorf("error making the access request: %w", err)
+	if authResponse.Status != 200 {
```

Comment:
Shouldn't we change 200 with the http.StatusOK constant ?

Reply:
done